platform/speed_formatted.cpp

```diff
@@ -0,0 +1,68 @@
+#include "speed_formatted.hpp"
+
+#include "platform/locale.hpp"
+#include "platform/localization.hpp"
+#include "platform/measurement_utils.hpp"
+
+#include "base/assert.hpp"
+
+namespace platform
+{
+using namespace measurement_utils;
+
+SpeedFormatted::SpeedFormatted(double speedMps) : SpeedFormatted(speedMps, GetMeasurementUnits()) {}
```

Comment:
It would be great to avoid introducing an implicit call to some external GetMeasurementUnits() method, and always pass it explicitly into the second constructor below.